sdk/formrecognizer/azure-ai-formrecognizer/samples/v3.1/async_samples/sample_convert_to_and_from_dict_v3_1_async.py

```diff
@@ -60,8 +60,8 @@ async def convert_to_and_from_dict_async():
     # save the dictionary as JSON content in a JSON file, use the AzureJSONEncoder
     # to help make types, such as dates, JSON serializable
     # NOTE: AzureJSONEncoder is only available with azure.core>=1.18.0.
-    with open('data.json', 'w') as f:
-        json.dump(recognized_form_dict, f, cls=AzureJSONEncoder)
+    with open('data.json', 'w') as f:  # type: ignore
+        json.dump(recognized_form_dict, f, cls=AzureJSONEncoder)  # type: ignore
```

Comment:
We should not have `type: ignore` in samples since we don't want to force our customers to have to use it. I think mypy is getting confused here since we use `f` in a difference context on L52-53. If you rename it to something else on these lines I think that will resolve your typing error and we won't need the ignore.